tiles/src/main/java/com/protomaps/basemap/layers/Buildings.java

```diff
@@ -20,10 +21,33 @@ public String name() {
     return "buildings";
   }
 
+  static int quantize_val(double val, int step) {
+    // special case: if val is very small, we don't want it rounding to zero, so
+    // round the smallest values up to the first step.
+    if( val < step ) {
+      return (int) step;
+    }
+
+    return (int) Math.round(val / step) * step;
+  }
+
   @Override
   public void processFeature(SourceFeature sf, FeatureCollector features) {
-    if (sf.canBePolygon() && (sf.hasTag("building") || sf.hasTag("building:part"))) {
+    if (sf.canBePolygon() && (
+            ( sf.hasTag("building") && !sf.hasTag("building", "no")) ||
```

Comment:
Sometimes `building` (and `building:part`) are marked `no`, we should exclude those from tiles.